app/uk/gov/hmrc/bindingtariffclassification/model/LiabilitySample.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+object LiabilitySample extends Enumeration {
+  type LiabilitySample = Value
+
+  val SENDING, NOT_SENDING = Value
```

Comment:
I still think we could reuse the existing sample status for this
Sending = some(awaiting)
Not sending = none

Reply:
From a conceptual point of view the principle is the same as with BTIs, the applicant is either sending a sample or they’re not